daemon/exec_linux_test.go

```diff
@@ -0,0 +1,53 @@
+// +build linux
+
+package daemon
+
+import (
+	"testing"
+
+	containertypes "github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/container"
+	"github.com/docker/docker/daemon/exec"
+	"github.com/gotestyourself/gotestyourself/assert"
+	"github.com/opencontainers/runc/libcontainer/apparmor"
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func TestExecSetPlatformOpt(t *testing.T) {
+	if !apparmor.IsEnabled() {
```

Comment:
For reference; looks like this check also checks for an `CONTAINER` env-var; https://github.com/moby/moby/blob/f58aa31075bf74ab8d2369dafb591ae43ed36ee6/vendor/github.com/opencontainers/runc/libcontainer/apparmor/apparmor.go#L11-L20

But our integration-cli checks in a different way; https://github.com/moby/moby/blob/f58aa31075bf74ab8d2369dafb591ae43ed36ee6/integration-cli/requirements_test.go#L110-L113